modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/rest/SW360AttachmentAwareClient.java

```diff
@@ -60,6 +71,28 @@ public T uploadAndAttachAttachment(T itemToModify, Path fileToAttach, String kin
         return uploadAndAttachAttachment(itemToModify, fileToAttach, requestEntity);
     }
 
+    protected boolean isAttachmentDuplicate(T itemToModify, SW360Attachment attachment, HttpHeaders header) {
```

Comment:
If we only compare by filename, this is not enough, default is that we overwrite the existing attachment instead of not uploading it.